src/App.ts

```diff
@@ -1301,7 +1301,7 @@ export default class App<AppCustomContext extends StringIndexed = StringIndexed>
 
     if (authorize === undefined && !usingOauth) {
       throw new AppInitializationError(
-        `${tokenUsage} \n\nSince you have not provided a token or authorize, you might be missing one or more required oauth installer options. See https://slack.dev/bolt-js/concepts/authenticating-oauth for these required fields.\n`,
+        `${tokenUsage} \n\nSince you have not provided a token or authorize, you might be missing one or more required oauth installer options. See https://tools.slack.dev/bolt-js/concepts/authenticating-oauth/ for these required fields.\n`,
```

Comment:
Love to see tests that are checking for this error code!

https://github.com/slackapi/bolt-js/blob/34b93abfa83f7e9e158a5b8706b706c9b6c3d541/test/unit/App/basic.spec.ts#L111-L119